src/main/java/org/sonarlint/intellij/ui/RecentProjectsPanel.kt

```diff
@@ -183,7 +183,7 @@ open class SonarLintRecentProjectPanel(private val onProjectSelected: (Project)
     }
 
     init {
-        var recentProjectActions = RecentProjectsManager.getInstance().getRecentProjectsActions(false, false).toList()
+        var recentProjectActions = RecentProjectListActionProvider.getInstance().getActions(false, false).toList()
```

Comment:
Looks like this class was only [introduced in 2020.1](https://github.com/JetBrains/intellij-community/commit/8c4e82f42915f98af74b72fba32ceb8f0c2e012c) 😕

Reply:
Indeed, the QA is red on 2019.3 axis. I will revert this change.